docs/CLI.md

```diff
@@ -108,6 +108,12 @@ If you want to inspect the cache, use `--showConfig` and look at the
 Runs tests related to the current changes and the changes made in the last
 commit. Behaves similarly to `--onlyChanged`.
 
+### `--changedSince`
```

Comment:
Can you add a "Since Jest 22.1.3+" or something?

Reply:
I said 22.2.0 because announcing that a feature is available since a patch release makes me uneasy. If it turns out that we release this feature without bumping the minor version then then `##### available in Jest **22.2.0+**` is not strictly a lie ;-)